packages/blockchain-wallet-v4-frontend/src/modals/Transactions/DownloadTransactions/index.tsx

```diff
@@ -122,36 +122,17 @@ const mapDispatchToProps = (dispatch: Dispatch, { coin }: OwnProps) => {
     clearTransactions: () =>
       dispatch(actions.core.data[coinCode].clearTransactionHistory()),
     fetchTransactions: (address, startDate, endDate) => {
-      if (coin === 'PAX') {
+      if (coin === 'PAX' || coin === 'USDT' || coin === 'WDGLD') {
```

Comment:
can we do a `coin in Erc20Enum` check?

Reply:
Sure but neither will be around for long.

Reply:
This file looks at coinfig now